packages/nuxt/README.md

```diff
@@ -119,11 +88,12 @@ if (process.env.SENTRY_DSN) {
 }
 ```
 
-The Nuxt runtime config does not work in the Sentry server to technical reasons (it has to be loaded before Nuxt is
-loaded). To be able to use `process.env` you either have to add `--env-file=.env` to your node command
+Using `useRuntimeConfig` does not work in the Sentry server config file due to technical reasons (the file has to be
+loaded before Nuxt is loaded). To be able to use `process.env` you either have to add `--env-file=.env` to your node
```

Comment:
```suggestion
loaded before Nuxt is loaded). To be able to use `process.env` you have to add `--env-file=.env` to your node
```

Reply:
The sentence continues after the code snippet with an "or", that's why there is an either :)

Reply:
Ah my bad, again